dark_emulator/darkemu/de_interface.py

```diff
@@ -1051,7 +1092,7 @@ def get_cosmology(self):
         Obtain the cosmological parameters currently set to the emulator.
 
         Returns:
-            numpy array: Cosmological parameters :math:`(\omega_b, \omega_{m}, \Omega_{de}, \ln(10^{10}A_s), n_s, w)`
+            numpy array: Cosmological parameters :math:`(\omega_b, \omega_{c}, \Omega_{de}, \ln(10^{10}A_s), n_s, w)`
```

Comment:
Same as above. It would be nice to change `\omega_{c}` to `\omega_c`.

Reply:
done!